### PR TITLE
Add Initiators & Deleters tutorial

### PR DESCRIPTION
**Pull request by sdhiscocks**
Tutorial for initiators and deleters.

### CODE REVIEW

File: docs/tutorials/08_Initiators_&_Deleters.py [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+
+# %%
+# Creating an Initiator
+# ---------------------
+# Here we are going to use a measurement based initiator, which will create a track from the
+# unassociated :class:`~.Detection` objects. This still requires a prior to be defined for the
+# :class:`~.Track`, but elements of the state vector that are measured are replaced by that of the
+# measurement, and the measurement's uncertainty (these defined by the
+# :class:`~.MeasurementModel`). In this example, as our sensor measures position, we only need to
+# be concerned about the values provided for the velocity and it's variance.
```
Comment (gawebb-dstl) [inline docs/tutorials/08_Initiators_&_Deleters.py:165]: This took a few reads to get my head around it. Could it be rewritten? 
```suggestion
# unassociated :class:`~.Detection` objects. A prior is still required to initialise the elements 
# of the state vector that cannot be initialised from the measurements and the measurement's 
# uncertainty (these defined by the :class:`~.MeasurementModel`). In this example, as our 
# sensor measures position, so the prior is needed to for the velocity and it's variance.
```
I don't think mine is any clearer 👎

File: docs/tutorials/08_Initiators_&_Deleters.py [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+# %%
+# Plot the resulting tracks.
+
+tracks_list = list(tracks)
+for track in tracks:
+    # Plot track.
+    ax.plot([state.state_vector[0, 0] for state in track],
+            [state.state_vector[2, 0] for state in track],
+            marker=".")
+fig
```
Comment (gawebb-dstl) [inline docs/tutorials/08_Initiators_&_Deleters.py:222]: Suggestion: Again a legend or key would useful

File: docs/tutorials/08_Initiators_&_Deleters.py [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+from stonesoup.types.groundtruth import GroundTruthPath, GroundTruthState
+
+start_time = datetime.now()
+truths = set()  # Truths across all time
+current_truths = set()  # Truths alive at current time
+
+transition_model = CombinedLinearGaussianTransitionModel([ConstantVelocity(0.005),
+                                                          ConstantVelocity(0.005)])
+
+for k in range(20):
```
[the remark this answers is not in the document]
Comment (sdhiscocks) [reply docs/tutorials/08_Initiators_&_Deleters.py:37]: This should probably be covered in earlier tutorials, as this is consistent across all of them. We'll look to make the change there.

File: docs/tutorials/08_Initiators_&_Deleters.py [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+    # Plot actual detections.
+    ax.scatter([state.state_vector[0] for state in set_ if isinstance(state, TrueDetection)],
+               [state.state_vector[1] for state in set_ if isinstance(state, TrueDetection)],
+               color='g')
+    # Plot clutter.
+    ax.scatter([state.state_vector[0] for state in set_ if isinstance(state, Clutter)],
+               [state.state_vector[1] for state in set_ if isinstance(state, Clutter)],
+               color='y',
+               marker='2')
+fig
```
[the remark this answers is not in the document]
Comment (sdhiscocks) [reply docs/tutorials/08_Initiators_&_Deleters.py:127]: Agree. Suggest we do this as one PR once all tutorials are are merged to keep sure all legends are the same.

File: docs/tutorials/08_Initiators_&_Deleters.py (hunk 1 above)
Comment (sdhiscocks) [reply docs/tutorials/08_Initiators_&_Deleters.py]: I've made an attempt to rewrite.

File: docs/tutorials/08_Initiators_&_Deleters.py (hunk 4 above)
[the remark this answers is not in the document]
Comment (sdhiscocks) [reply docs/tutorials/08_Initiators_&_Deleters.py]: I've created #241 so we don't forget.

**Pull request merged**